pypeit/spectrographs/gemini_gmos.py

```diff
@@ -453,8 +453,11 @@ def get_mosaic_par(self, mosaic, hdu=None):
             msc_rot[i] = rotation[d]
             msc_tfm[i] = build_image_mosaic_transform(shape, msc_sft[i], msc_rot[i], binning)
 
+        # Hard coded for now. Order of the interpolation used to construct the mosaic
+        msc_order = 0
```

Comment:
maybe put this in the `def` 

Is an easier spot to find (and then you can comment it there)